test/integration/edition_workflow_test.rb

```diff
@@ -610,6 +610,26 @@ class EditionWorkflowTest < JavascriptIntegrationTest
     assert page.has_css?(".btn.btn-large.btn-primary", text: "Publish now")
   end
 
+  test "Welsh editors cannot see review buttons for non-Welsh editions" do
+    edition = FactoryBot.create(:edition, :in_review, panopticon_id: FactoryBot.create(:artefact).id)
+    login_as("WelshEditor")
+
+    visit_edition edition
+
+    assert_not page.has_link?("Needs more work", class: %w[btn btn-info add-top-margin])
+    assert_not page.has_link?("OK for publication", class: %w[btn btn-info add-top-margin])
+  end
+
+  test "Welsh editors can see review buttons for Welsh editions" do
+    edition = FactoryBot.create(:edition, :in_review, panopticon_id: FactoryBot.create(:artefact, :welsh).id)
+    login_as("WelshEditor")
+
+    visit_edition edition
+
+    assert page.has_link?("Needs more work", class: %w[btn btn-info add-top-margin])
+    assert page.has_link?("OK for publication", class: %w[btn btn-info add-top-margin])
```

Comment:
I don't think we need to specify the classes here. We might tweak the style of the buttons slightly, and that shouldn't break these tests. Just searching for the link (and ideally the link's destination) is sufficient.

Reply:
That's true! I was trying to be specific but `class` isn't the best way to do that - `href` is much better.